### PR TITLE
refactor: updated README file to reflect template changes

### PR DESCRIPTION
**Pull request by Mashal-m**
### Description

The README of the MFEs needs to be updated because the frontend app template has been updated.

Issue related: https://github.com/openedx/wg-frontend/issues/169

### CODE REVIEW

**Review by AfaqShuaib09 — APPROVED** [review]

**Comment by Mashal-m** [comment]
@AfaqShuaib09 could you please merge this as well, I do not have permission.

**Comment by AfaqShuaib09** [comment]
> @AfaqShuaib09 could you please merge this as well? I do not have permission.

Hey @Mashal-m, I'm waiting for 2nd approval on your PR. I will merge it once it receives the second review.
In the meantime, Could you please rebase your brach with master? Thank you

**Pull request merged**
